test/fixtures/test-runner/output/typescript-coverage.snapshot

```diff
@@ -29,11 +29,11 @@ ok 1 - foo
 #  fixtures                   |        |          |         | 
 #   test-runner               |        |          |         | 
 #    coverage                 |        |          |         | 
-#     bar.mts                 |   0.00 |   100.00 |  100.00 | 1-3
+#     bar.mts                 |  33.33 |   100.00 |    0.00 | 2-3
 #     foo.mts                 | 100.00 |   100.00 |  100.00 | 
 #    output                   |        |          |         | 
 #     typescript-coverage.mts | 100.00 |   100.00 |  100.00 | 
 # ----------------------------------------------------------------------------
-# all files                   |  85.29 |   100.00 |   85.71 | 
+# all files                   |  93.55 |   100.00 |   85.71 | 
```

Comment:
Since `bar.mts` is imported before the mock in `typescript-coverage.ts`, that one line should be covered.